packages/main/src/Breadcrumbs.js

```diff
@@ -0,0 +1,534 @@
+import ItemNavigation from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import {
+	isSpace,
+	isShow,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import NavigationMode from "@ui5/webcomponents-base/dist/types/NavigationMode.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import BreadcrumbsDesign from "./types/BreadcrumbsDesign.js";
+import BreadcrumbsSeparatorStyle from "./types/BreadcrumbsSeparatorStyle.js";
+import BreadcrumbsItem from "./BreadcrumbsItem.js";
+import {
+	BREADCRUMBS_ARIA_LABEL,
+	BREADCRUMBS_OVERFLOW_ARIA_LABEL,
+	BREADCRUMBS_CANCEL_BUTTON,
+} from "./generated/i18n/i18n-defaults.js";
+import Link from "./Link.js";
+import Label from "./Label.js";
+import ResponsivePopover from "./ResponsivePopover.js";
+import Popover from "./Popover.js";
```

Comment:
Do you need the Popover as well, I saw the Responsive Popover in use, but the the pure Popover I did not.

Reply:
Removed
(thought it was needed as I saw it in the ui5-select dependencies list, but now I see that ui5-select needs it for its own reason, while here is redundant)
Thnx